paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc

```diff
@@ -72,9 +72,25 @@ class FCPrimitiveFactory {
     auto dst_desc = CreateMemDescriptor<T_out>(output, MKLDNNMemoryFormat::any);
 
     fc_ = CreateFcPrimitive(*input_, *weights_, dst_desc, bias, output, ctx);
+    this->Execute();
```

Comment:
The idea behind this factory was to return an FC primitive and feed it to the stream from the outer code. Now that it can no longer be done without accessing the memory primitives, I suggest that this method would no longer return `inner_product_forward` but be of the void return type. 

At the same time, I'd move the call to Execute() to Compute function. The way it is implemented right now in this PR, the FC would only execute once, upon its creation. On the subsequent runs, it'd only enter the part starting on line 52
```C++
    if (fc_) {
      UpdateDataPointers(ctx, output, input);
      return *fc_;
    }
```
...to update the pointers, but without executing the primitive.

Reply:
Sure, obvious mistake. I will fix it next commit